mica/archive/aca_l0.py

```diff
@@ -206,11 +207,13 @@ def __init__(self, msids, start, stop):
 
 
 def obsid_times(obsid):
-    aca_db = dict(dbi="sybase", server="sybase", user="aca_read")
-    with Ska.DBI.DBI(**aca_db) as db:
-        obspars = db.fetchall("""select * from obspar where obsid = %d
-                                 order by obi""" % obsid)
-    return obspars[0]['tstart'], obspars[0]['tstop']
+    from kadi import events  # Kadi is a big import so defer
+    dwells = events.dwells.filter(obsid=obsid)
+    n_dwells = len(dwells)
+    tstart = dwells[0].tstart
+    tstop = dwells[n_dwells - 1].tstop
```

Comment:
I don't recall what obsid_times is doing anyway.  Seems like it should have been reading from the mica obspar table if it really cared about obspars.